users/urls.py

```diff
@@ -1,7 +1,10 @@
+from django.conf.urls import url
 from django.urls import path
 
-from users.views import SignupView
+from users.views import GetView, SignupView, UpdateView
 
 urlpatterns = [
     path("signup/", SignupView.as_view()),
+    url(r"^(?P<username>\w+)/edit/$", UpdateView.as_view(), name="update"),
```

Comment:
We usually have one endpoint for getting, updating and removing stuff. Just use different HTTP methods for different actions

Reply:
Please use user id instead of username in the url

Reply:
Done